nixos/tests/docker-tools.nix

```diff
@@ -1,6 +1,49 @@
 # this test creates a simple GNU image with docker tools and sees if it executes
 
-import ./make-test-python.nix ({ pkgs, ... }: {
+import ./make-test-python.nix ({ pkgs, ... }:
+let
+  # nixpkgs#214434: dockerTools.buildImage fails to unpack base images
+  # containing duplicate rootfs diffs when those duplicate tarballs
+  # appear under the manifest's 'Layers'. Docker can generate images
+  # like this even though dockerTools does not.
+  repeatedLayerTestImage =
```

Comment:
It looks like all other images involved in our tests come from `dockerTools`' `examples.nix`. I decided to define the image here instead because it's awkward and seems like a poor example.

If there's a better place for this to go, I'd be happy to move it.

Reply:
This is the right thing to do. We could probably move more bad examples. (but not in this pr :) )